### PR TITLE
Changes the position of the parameter enableExternalVideo in the settings.yml

### LINKED ISSUE #7769: Tweak parameter to disable Share YouTube (for consistency)
In settings.yml add

```
app:
  ...
  note:
    ...
  externalVideoPlayer:
    enabled: true
```

remove app.enableExternalVideo: true

And make adjustments in 
Screenshot from 2019-07-12 11-41-38

The goal is to make this setting more consistent with how we enable/disable other features like Notes, Closed Captions, etc

### PR DESCRIPTION
**Pull request by diegobenetti**
Closes #7769

### CODE REVIEW

File: bigbluebutton-html5/private/config/settings.yml [hunk 1]
```diff
@@ -78,11 +78,12 @@ public:
       displayBrandingArea: false
     connectionTimeout: 60000
     showHelpButton: true
-    enableExternalVideo: true
     effectiveConnection:
       - critical
       - danger
       - warning
+  enableExternalVideo:
```
Comment (antobinary) [inline bigbluebutton-html5/private/config/settings.yml]:
```
externalVideoPlayer:
  enabled: true
```

Comment (diegobenetti) [reply bigbluebutton-html5/private/config/settings.yml]: oh, right! I've just fixed it.

File: bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ class ActionsBar extends PureComponent {
     } = recordSettingsList;
 
     const actionBarClasses = {};
-    const { enableExternalVideo } = Meteor.settings.public.app;
+    const { enabled: enableExternalVideo } = Meteor.settings.public.externalVideoPlayer;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx]:
```suggestion
    const { enabled: allowExternalVideo } = Meteor.settings.public.externalVideoPlayer;
```

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx]: so you can simplify a few lines down

`- allowExternalVideo: enableExternalVideo,`
`+ allowExternalVideo,`

**Pull request merged**